### ISSUE TITLE
Parse error with Java import pattern

### ISSUE THREAD

**Issue by mschwager** [semgrep/semgrep#866]
Rule file:

```yaml
rules:
- id: test-import
  pattern: import java.io.File;
  message: "import test"
  languages: [java]
  severity: ERROR
```

Java file:

```java
import java.io.File;  // Import the File class
import java.io.FileNotFoundException;  // Import this class to handle errors
import java.util.Scanner; // Import the Scanner class to read text files

public class ReadFile {
  public static void main(String[] args) {
    try {
      File myObj = new File("filename.txt");
      Scanner myReader = new Scanner(myObj);
      while (myReader.hasNextLine()) {
        String data = myReader.nextLine();
        System.out.println(data);
      }
      myReader.close();
    } catch (FileNotFoundException e) {
      System.out.println("An error occurred.");
      e.printStackTrace();
    }
  }
}
```

`semgrep` output:

```
$ semgrep --config conf.yaml test.java 
invalid pattern "import java.io.File;": Parsing.Parse_error
```

**Comment by mschwager** [comment]
More specifically, I'd like to be able to specify the pattern `java.io.$ANYTHING;`, similar to Python, and catch this code `File("filename.txt");`. Note that this also gives a parse error:

```
$ semgrep --lang java --pattern 'java.io.$ANYTHING;' test.java 
invalid pattern "java.io.$ANYTHING;": Parsing.Parse_error
```

**Comment by nbrahms** [comment]
Implemented in 0.10.

Note that you'll need to specify an import statement here:
```
import javax.crypto.$ANYTHING;
```
